apps/openmw-mp/CMakeLists.txt

```diff
@@ -136,7 +136,7 @@ target_compile_definitions(tes3mp-server PRIVATE $<$<CONFIG:Debug>:SERVER_DEBUG>
 
 set_property(TARGET tes3mp-server PROPERTY CXX_STANDARD 14)
 if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-    target_compile_options(tes3mp-server PRIVATE -Wno-ignored-qualifiers -ftemplate-depth=2048)
+    target_compile_options(tes3mp-server PRIVATE -Wno-ignored-qualifiers -ftemplate-depth=4096)
```

Comment:
Visual Studio template depth is limited to 2048. https://docs.microsoft.com/en-GB/cpp/cpp/compiler-limits
So this is not valid fix.

Reply:
The conditions in the if statement should prevent this being used with Visual Studio. This is required for the g++ build.

Reply:
For what it's worth, the sol2 documentation says template depth is an [issue](https://sol2.readthedocs.io/en/latest/errors.html?highlight=template%20depth) on older versions of clang and gcc.

Reply:
I am not sure I understand what you want me to do. I think you just want me to remove the fix, so that is what I will do. Unless something else has changed, this will break the gcc build.

Reply:
Ideally someone would set up AppVeyor to test Windows builds.

Reply:
I'll fix GCC build later